service/http.proxy/src/test/java/org/kaazing/gateway/service/http/proxy/HttpProxyPersistenceIT.java

```diff
@@ -31,18 +36,13 @@
 import org.kaazing.gateway.server.test.config.builder.GatewayConfigurationBuilder;
 import org.kaazing.k3po.junit.annotation.Specification;
 import org.kaazing.k3po.junit.rules.K3poRule;
-
-import java.net.URI;
-
-import static java.util.concurrent.TimeUnit.SECONDS;
-import static org.junit.rules.RuleChain.outerRule;
+import org.kaazing.test.util.MethodExecutionTrace;
 
 public class HttpProxyPersistenceIT {
 
-    @Rule
-    public TestRule timeout = new DisableOnDebug(new Timeout(10, SECONDS));
-
-    private final K3poRule robot = new K3poRule();
+    private final TestRule timeout = new DisableOnDebug(new Timeout(10, SECONDS));
+    private final TestRule testExecutionTrace = new MethodExecutionTrace();
```

Comment:
Suggest calling this `trace` instead. Btw, do we still need it or was it just useful during test authorship?